RadixWallet/Features/DappInteractionFeature/Children/AccountPermissionChooseAccounts/AccountPermissionChooseAccounts+Reducer.swift

```diff
@@ -167,9 +168,7 @@ struct AccountPermissionChooseAccounts: Sendable, FeatureReducer {
 
 				var accountsLeftToVerifyDidSign: Set<Account.ID> = Set(selectedAccounts.map(\.id))
 				let walletAccountsWithProof: [P2P.Dapp.Response.Accounts.WithProof] = signedAuthChallenge.entitySignatures.map {
-					guard case let .account(account) = $0.signerEntity else {
-						fatalError()
-					}
+					let account = try! $0.signerEntity.asAccount()
```

Comment:
this `fatalError` -> `try!` ought to be reworked ofc... outside scope of Profile integration.